docs/tutorial-get.rst

```diff
@@ -69,6 +75,16 @@ This will return data objects with OK status, created in October 2018, order
 them by descending modified date and return first 3 objects. Quite powerful
 isn't it?
 
+.. note::
+
+  For a complete list of filtering options use a "wrong" filtering
```

Comment:
Keep the lines as long as possible while not exceeding 79 characters, i.e., put "argument" in this line.